docs/DeveloperGuide.adoc

```diff
@@ -297,33 +298,33 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new task | check the details of the task when I want
```

Comment:
User vs Tutor: Do they refer to the same type of user? Would appreciate a bit more details with respect to the differences between these two types of users.

Reply:
Task: Perhaps a bit more detail with respect to what a task refers to, currently we're not sure what a task is.